src/main/java/browserview/BrowserComponent.java

```diff
@@ -486,4 +486,9 @@ public String getCurrentUrl() {
         return driver.getCurrentUrl();
     }
 
+    public void loadTab(String currentUrl) {
+        if (isCurrentUrlIssue() && (currentUrl.endsWith("/commits") || currentUrl.endsWith("/files"))) {
```

Comment:
The `endsWith` part can be factored out into `GitHubURL`.

Also I don't think we need to pass the URL as a parameter, given that all this method does is switch to the discussion tab. Conceptually it's not really parameterised by the current URL, and being a `BrowserComponent` method, can already access it.